RestaurantReservation.Db/MenuItem.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace RestaurantReservation.Db
+{
+    public class MenuItem
+    {
+        public int ItemId { get; set; }
+        public int RestaurantsId { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public float Price { get; set; }
```

Comment:
For precision concerns especially when dealing with MS SQL server, avoid using **float** data type, use **double** instead

Reply:
sry, my bad, I mean **decimal** not double or float